app/views/home/index.rb

```diff
@@ -4,6 +4,19 @@ def page_title
       'Europeana Collections'
     end
 
+    def js_slsb_ld
+      {
+        "@context": "http://schema.org",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.